CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# Changelog
+
+[npm history][1]
+
+[1]: https://www.npmjs.com/package/@google-cloud/bigquery-data-transfer?activeTab=versions
+
+## v0.4.0
+
+### Implementation Changes
+- Re-generate library using /synth.py (#51)
```

Comment:
This doesn't tell the user what changed :) Can you take a look at the PR, and see what actually happened?